src/mercury_engine_data_structures/formats/bmses.py

```diff
@@ -0,0 +1,46 @@
+import functools
+
+import construct
+from construct.core import (
+    Const,
+    Construct,
+    Float32l,
+    Hex,
+    Int32sl,
+    Int32ul,
+    Struct,
+)
+
+from mercury_engine_data_structures.common_types import CVector3D, StrId, make_vector
+from mercury_engine_data_structures.formats import BaseResource
+from mercury_engine_data_structures.game_check import Game
+
+BMSES = Struct(
+    "magic" / Const(b"MSES"),
+    "version" / Hex(Int32ul),
```

Comment:
We usually use a const here and write down the value. (Dunno why `BMSBK` did it different).